internal/server/worker_auth.go

```diff
@@ -96,7 +122,7 @@ func (w *WorkerAuth) ValidateNewWorkerAuth(ctx context.Context) error {
 	if w.WorkerEncryptionPubKey == nil {
 		return errors.New(ctx, errors.InvalidParameter, op, "missing WorkerEncryptionPubKey")
 	}
-	if w.ControllerEncryptionPrivKey == nil {
+	if w.CtControllerEncryptionPrivKey == nil {
```

Comment:
Ditto

Reply:
this one is definitely the case, as this is the validate function, it's called before storing the value in the database. since we're not storing the plaintext field, it doesn't need to be checked!

Reply:
(I will, however, update the error string, as it's somewhat misleading as-is)